tests/e2e/runtime/runtime_test.go

```diff
@@ -74,10 +74,10 @@ func TestMain(m *testing.M) {
 			Name:     "runtime-bindings-http",
 			TypeName: "bindings.kafka",
 			MetaData: map[string]string{
-				"brokers":       "dapr-kafka:9092",
-				"topics":        "runtime-bindings-http",
-				"consumerGroup": "group1",
-				"authRequired":  "false",
+				"brokers":       `"dapr-kafka:9092"`,
+				"topics":        `"runtime-bindings-http"`,
+				"consumerGroup": `"group1"`,
```

Comment:
I see all value has quotations. Can we use one of values without quotations? just for reference.

Reply:
The Kubernetes Client requires that for strings are quoted for runtime.JSON values.